api/errors.py

```diff
@@ -3,8 +3,8 @@
 
 def unauthorized(message):
     ''' User is unauthorised to perform action.'''
-    body = {'status': 401, 'error': 'unauthorized', 'message': message}
-    return Response(body, status=status.HTTP_401_UNAUTHORIZED)
```

Comment:
The purpose of returning stuffs via the `Response` object is so that postman or whatever client we use can recognize the status code. 
Is there any reason why you removed it here?

Reply:
@andela-sjames I decided to keep the file in case we choose to use it later. I feel there might be a need for it. We can delete it later if we don't need it

Reply:
@BolajiOlajide For now, we are making use of APIException class from DRF exceptions.py so the file is not being used. I made the change when I was trying to make use of the file.